### PR TITLE
Enable compose project names

### LINKED ISSUE #85: Enable docker-compose project names
See https://github.com/research-software-directory/RSD-as-a-service/pull/430

### PR DESCRIPTION
**Pull request by Normo**
* Closes #85

### CODE REVIEW

File: templates/docker-compose.yml.j2 [hunk 1]
```diff
@@ -8,7 +8,6 @@ version: "3.0"
 
 services:
   database:
-    container_name: database
```
Comment (tobiashuste) [inline templates/docker-compose.yml.j2:11]: Are the containers still named with a useful name. Probably in this case it's derived from `database`? Right?

File: handlers/main.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 - name: "Restart nginx"
   community.docker.docker_compose:
-    project_name: "rsd"
+    project_name: "{{ rsd_compose_project_name | default('rsd') }}"
```
Comment (tobiashuste) [inline handlers/main.yml:20]: I could imagine it's beneficial to specify `rsd_compose_project_name` in `defaults/main.yml`. This way, we would not need to adapt multiple lines of code if we were to change the default name. WDYT?

**Review by tobiashuste — COMMENTED** [review]
Thanks, @Normo. This looks fine from my side.

I added one suggestion that could be considered.

File: templates/docker-compose.yml.j2 (hunk 1 above)
Comment (Normo) [reply templates/docker-compose.yml.j2:11]: The got generated names according to the pattern: <PROJECT NAME>_<SERVICE NAME>_<COUNTER>

File: handlers/main.yml (hunk 2 above)
Comment (Normo) [reply handlers/main.yml]: You're right. No idea what got me into this. Changed by 8b338f6875e05a9a6010347045aa1b0038880e2f

**Review by tobiashuste — APPROVED** [review]
Thanks, @Normo. This looks good to me now. Feel free to merge :boat:

**Pull request merged**